### PR TITLE
Replace events with teaching events

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

https://trello.com/c/BqqVPfpe/3332-deploy-new-events-design-to-production

### Context

Started making the move, bulk is 'done' but it needs:

* [x] the remains of old event functionality removing
* [x] remove noindex from new event pages
* [x] remove canonical tags from new event pages
* [x] track down any last remnants of old events views (this is done in #2620)
* [x] update backlinks / copy round the site
* [ ] some final checks

### CODE REVIEW

**Comment by peteryates** [comment]
The `event-box` is used on the first page of the signup - that was one of the causes of woe in my first attempt at this on Friday 🙈 

Definitely worth a refactor and reorganise once this is in though. Still just working through and seeing what's removable. Good call on the events category redirects, I'll add them in - I think just sending them to `/events` is fine.

**Comment by gemmadallmandfe** [comment]
@peteryates In terms of the category pages, I think we need the following (they are all going to 404s at the moment)

* `/event-categories/train-to-teach-events` => `/events/about-ttt-events`

* `/event-categories/online-q-as` => `/events` (for now - to be replaced by pointing to pre-filtered listings once the URLs are simplified)

* `/event-categories/school-and-university-events` => `/events` (for now - to be replaced by pointing to pre-filtered listings once the URLs are simplified)

**Comment by gemmadallmandfe** [comment]
> We need to check backlinks and copy round the site @EllieNodder @peteryates
> 
> For example:
> 
> * we talk about 'School and university events' on `/help-and-advice` but we are changing this terminology to be 'Training provider events', so that will need updating
>   @EllieNodder could you possibly add this change into this PR?
> * several backlinks to online Q&A event category page (`/event-categories/online-q-as`) - this page is disappearing so we need to change to pre-filtered online Q&A events (might be easier to do this after we've simplified the URLs @peteryates - let's discuss?)
> * backlink to TTT page (changing URL from `/event-categories/train-to-teach-events`  to `\events/about-ttt-events`)
>   @EllieNodder is this something you could possibly look at? But let's discuss first
> 
> I may have missed something - will review in detail tomorrow. @EllieNodder I'd also value your eagle eyes on this!

**Update**

* I have changed the Help and advice page

* Online Q&A links will redirect to the events main page for the time being - we can update to pre-filtered results once the URL simplification has been done

* I have changed backlinks to the About TTT page
It may be more appropriate to change some of these links to pre-filtered results after we've simplified the URLs post-deploy - I will note this on the simplification ticket

Stand down @EllieNodder !

File: app/views/content/blog/getting-ready-to-apply.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Something that many teacher trainer providers will be looking for in an applicat
 
 ## 8. Get some advice at an event
 
-Our Train to Teach events will provide you with a wealth of information and help you turn questions to confidence on your journey to the classroom. At some of our Train to Teach events you can meet a whole range of local training providers; at others you’ll have the chance to put your questions to a panel of experts. Some events are in person and others replicate the experience online. [Book your place](/event-categories/train-to-teach-events).
+Our Train to Teach events will provide you with a wealth of information and help you turn questions to confidence on your journey to the classroom. At some of our Train to Teach events you can meet a whole range of local training providers; at others you’ll have the chance to put your questions to a panel of experts. Some events are in person and others replicate the experience online. [Book your place](/events/about-ttt-events).
```
Comment (gemmadallmandfe) [inline app/views/content/blog/getting-ready-to-apply.md:77]:
```suggestion
Our Train to Teach events will provide you with a wealth of information and help you turn questions to confidence on your journey to the classroom. At some of our Train to Teach events you can meet a whole range of local training providers; at others you’ll have the chance to put your questions to a panel of experts. Some events are in person and others replicate the experience online. [Book your place at a Train to Teach event](/events/about-ttt-events).
```

File: app/views/content/blog/5-reasons-to-attend-a-train-to-teach-event.md [hunk 2]
```diff
@@ -55,4 +55,4 @@ You can have a one-to-one chat with a current teacher who can provide you with i
 
 They can also talk about why they love teaching and how rewarding it is, giving you a balanced view and insight into what the life of a teacher is really like.
 
-[Find a Train to Teach event near you.](/event-categories/train-to-teach-events)
+[Find a Train to Teach event near you.](/events/about-ttt-events)
```
Comment (gemmadallmandfe) [inline app/views/content/blog/5-reasons-to-attend-a-train-to-teach-event.md:58]:
```suggestion
[Find a Train to Teach event near you](/events/about-ttt-events).
```

**Comment by gemmadallmandfe** [comment]
@peteryates Google maps embed doesn't seem to be working

image

This is also true in production on the `/teaching-events` pages - but it is fine in prod on `/events`

Think we just need to sort this before pushing this PR to prod

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**